src/screens/Settings/components/WineSettings/index.tsx

```diff
@@ -189,6 +193,18 @@ export default function WineSettings({
           ))}
         </select>
       </span>
+      {wineVersion.name.includes('CrossOver') && <span className="setting">
+        <span className="settingText">{t('setting.winecrossoverbottle', 'CrossOver Bottle')}</span>
```

Comment:
I think t('setting.winecrossoverbottle', 'CrossOver Bottle') needs to be set in the translation.json for each language. Just insert the english version for each language. People on webplate will translate it later.
Or if i'm wrong here @flavioislima ?

Reply:
You're wrong here :P
Actually, we should not touch the translations file. Just add the new key and then run: ´yarn i18n` and it will be created for all languages. Otherwise, we will have conflicts with the weblate branch.

Reply:
Ok good to know :).